app/assets/stylesheets/common/base/header.scss

```diff
@@ -201,34 +201,6 @@
   .notifications {
     position: relative;
   }
-  .ring {
```

Comment:
Farewell, `.ring`! We'll never forget the joy you brought us 😅💍

Reply:
💍 rinGATE shall never be forgotten